README.md

```diff
@@ -24,3 +24,59 @@ See `const thumber` in `scthumbd.js` for now...
 ```
 npm run scthumbd
 ```
+
+
+## Envrionment variables
+
+`PORT` - listening port, defaults to 4001.
+
+`WORKERS` - number of worker processes to spawn, defaults to number of cpus detected.
+
+
+## Docker
+
+This assumes Docker Compose V2 is used - replace `docker compose` with `docker-compose` if using Docker Compose V1.
+
+Running for development:
+```
+docker compose up
+```
+will mount the local directory to the container; code changes will be reflected when the contianer is restarted.
+The container will accept requests on port 4001. To use a different port, provide a `HTTP_PORT` env, e.g.:
+
+```
+HTTP_PORT=8001 docker compose up
+```
+
+To override settings inside the container such as the number of worker or listening port, it is recommended to create another docker-compose file with the extra settings and use that file when running `docker compose`, e.g.:
+
+```
+docker compose -f docker-compose.yml -f docker-compose.dev.yml up
+```
+
+
+To just build an image
+```
+docker build -t <repository_name>:<tag> .
+```
+
+### Networking Notes
```

Comment:
These instructions rely on osu-web being exposed to `0.0.0.0` or whatever is Docker's default.

I think a better alternative would be to spawn scthumber in an existing osu-web Docker deployment instead, by referencing its network by name. See: https://docs.docker.com/compose/networking/#use-a-pre-existing-network

Reply:
It assumes this is run standalone; networking, remote hosts, build customizations, running outside Docker Desktop aren't part of the scope.